### PR TITLE
docs: Add bi-weekly report 0523-0605

### PR DESCRIPTION
**Pull request by Qizeng-api7**
Changes:

Add bi-weekly report 0523-0605 in en-us and zh-cn.

### CODE REVIEW

File: blog/en/blog/2023/06/08/weekly-report.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: "Biweekly Report (May 23 - Jun 8)"
+keywords: 
+- Apache APISIX
+- API Gateway
+- Weekly Report
+- Contributor
+description: The Apache APISIX Community Biweekly Report can help community members better grasp the progress of the Apache APISIX community so that everyone can participate in the Apache APISIX community.
+tags: [Community]
+image: https://static.apiseven.com/uploads/2023/04/26/vh2yfEWf_%E7%A4%BE%E5%8C%BA%E5%8F%8C%E5%91%A8%E6%8A%A5-%E5%85%AC%E4%BC%97%E5%8F%B7%E5%A4%B4%E5%9B%BE-%E8%8B%B1%E6%96%8720230426.png
```
Comment (Yilialinn) [inline blog/en/blog/2023/06/08/weekly-report.md]: wrong date

File: blog/en/blog/2023/06/08/weekly-report.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+> From 5.23 to 6.05, 20 contributors submitted 37 commits for Apache APISIX. Thank you for your contributions to Apache APISIX.
+<!--truncate-->
+
+## Introduction
+
+Apache APISIX grew up as a community from the first day it was open-sourced, and quickly became the most active open-source API gateway project in the world. These achievements are inseparable from the joint efforts of community partners.
+
+"If you want to go fast, go alone. If you want to go far, go together." The Apache APISIX Community Biweekly Report can help community members better grasp the progress of the Apache APISIX community so that everyone can participate in the Apache APISIX community.
+
+We have also sorted out some issues for newcomers to the community to participate in! If you are interested, don't miss it!
```
Comment (Yilialinn) [inline blog/en/blog/2023/06/08/weekly-report.md]: I see no good issues below

File: blog/en/blog/2023/06/08/weekly-report.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+---
+title: "Biweekly Report (May 23 - Jun 8)"
```
Comment (Yilialinn) [inline blog/en/blog/2023/06/08/weekly-report.md]:
```suggestion
title: "Biweekly Report (May 23 - June 8)"
```

File: blog/en/blog/2023/06/08/weekly-report.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+---
+title: "Biweekly Report (May 23 - June 8)"
```
Comment (yzeng25) [inline blog/en/blog/2023/06/08/weekly-report.md]:
```suggestion
title: "Biweekly Report (May 23 - June 5)"
```

File: blog/zh/blog/2023/06/08/weekly-report.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+![贡献者名单](https://security.feishu.cn/link/safety?target=https%3A%2F%2Fstatic.apiseven.com%2Fuploads%2F2023%2F06%2F08%2FG1yIQjyj_%25E6%2589%2580%25E6%259C%2589%25E8%25B4%25A1%25E7%258C%25AE%25E8%2580%2585%25E6%25B5%25B7%25E6%258A%25A5.png&scene=ccm&logParams=%7B%22location%22%3A%22ccm_default%22%7D&lang=zh-CN)
+
+![新晋贡献者](https://static.apiseven.com/uploads/2023/06/08/MOxcew0f_%E6%96%B0%E6%99%8B%E8%B4%A1%E7%8C%AE%E8%80%85%E6%B5%B7%E6%8A%A5.png)
+
+## 近期亮点功能
+
+### Apache APISIX
+
+- [使用单个 http 长链接 watch 所有资源](https://github.com/apache/apisix/pull/9456)（贡献者：[kingluo](https://github.com/kingluo)）
+
```
Comment (yzeng25) [inline blog/zh/blog/2023/06/08/weekly-report.md]:
```suggestion
## 近期亮点功能

- [使用单个 http 长链接 watch 所有资源](https://github.com/apache/apisix/pull/9456)（贡献者：[kingluo](https://github.com/kingluo)）

```

File: blog/en/blog/2023/06/08/weekly-report.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+![Apache APISIX Contributors List](https://security.feishu.cn/link/safety?target=https%3A%2F%2Fstatic.apiseven.com%2Fuploads%2F2023%2F06%2F08%2FG1yIQjyj_%25E6%2589%2580%25E6%259C%2589%25E8%25B4%25A1%25E7%258C%25AE%25E8%2580%2585%25E6%25B5%25B7%25E6%258A%25A5.png&scene=ccm&logParams=%7B%22location%22%3A%22ccm_default%22%7D&lang=zh-CN)
+
+![Apache APISIX New Contributors](https://static.apiseven.com/uploads/2023/06/08/MOxcew0f_%E6%96%B0%E6%99%8B%E8%B4%A1%E7%8C%AE%E8%80%85%E6%B5%B7%E6%8A%A5.png)
+
+## Highlights of Recent Features
+
+### Apache APISIX
+
+- [Use a single http long connection to watch all resources](https://github.com/apache/apisix/pull/9456) (Contributor: [kingluo](https://github.com/kingluo))
```
Comment (yzeng25) [inline blog/en/blog/2023/06/08/weekly-report.md]:
```suggestion
## Highlights of Recent Features

- [Use a single http long connection to watch all resources](https://github.com/apache/apisix/pull/9456) (Contributor: [kingluo](https://github.com/kingluo))
```

**Comment by yzeng25** [comment]
**Please merge this before https://github.com/apache/apisix-website/pull/1612.**

If merge https://github.com/apache/apisix-website/pull/1612 before this, it will create merge conflicts, so just don't.

File: blog/en/blog/2023/06/08/weekly-report.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+## Introduction
+
+Apache APISIX grew up as a community from the first day it was open-sourced, and quickly became the most active open-source API gateway project in the world. These achievements are inseparable from the joint efforts of community partners.
+
+"If you want to go fast, go alone. If you want to go far, go together." The Apache APISIX Community Biweekly Report can help community members better grasp the progress of the Apache APISIX community so that everyone can participate in the Apache APISIX community.
+
+## Contributor Statistics
+
+![Apache APISIX Contributors List](https://security.feishu.cn/link/safety?target=https%3A%2F%2Fstatic.apiseven.com%2Fuploads%2F2023%2F06%2F08%2FG1yIQjyj_%25E6%2589%2580%25E6%259C%2589%25E8%25B4%25A1%25E7%258C%25AE%25E8%2580%2585%25E6%25B5%25B7%25E6%258A%25A5.png&scene=ccm&logParams=%7B%22location%22%3A%22ccm_default%22%7D&lang=zh-CN)
```
Comment (Yilialinn) [inline blog/en/blog/2023/06/08/weekly-report.md]:
```suggestion
![Apache APISIX Contributors List](https://static.apiseven.com/uploads/2023/06/09/XavQu9xL_contributor%20list0609.png)
```

Comment (Yilialinn) [reply blog/en/blog/2023/06/08/weekly-report.md]: Please align the names to make the picture look more neat! You can use this one this time.

File: blog/zh/blog/2023/06/08/weekly-report.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+## 导语
+
+Apache APISIX 从开源第一天就以社区方式成长，迅速成为全世界最活跃的开源 API 网关项目。这些成就，离不开社区小伙伴们的共同奋斗。
+
+“独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的进展，方便大家参与到 Apache APISIX 社区中来。
+
+## 贡献者统计
+
+![贡献者名单](https://security.feishu.cn/link/safety?target=https%3A%2F%2Fstatic.apiseven.com%2Fuploads%2F2023%2F06%2F08%2FG1yIQjyj_%25E6%2589%2580%25E6%259C%2589%25E8%25B4%25A1%25E7%258C%25AE%25E8%2580%2585%25E6%25B5%25B7%25E6%258A%25A5.png&scene=ccm&logParams=%7B%22location%22%3A%22ccm_default%22%7D&lang=zh-CN)
```
Comment (Yilialinn) [inline blog/zh/blog/2023/06/08/weekly-report.md]:
```suggestion
![贡献者名单](https://static.apiseven.com/uploads/2023/06/09/XavQu9xL_contributor%20list0609.png)
```

Comment (Yilialinn) [reply blog/zh/blog/2023/06/08/weekly-report.md]: Ditto

**Review by yzeng25 — APPROVED** [review]

**Review by guoqqqi — APPROVED** [review]

**Pull request merged**